source/guide_alertmanager.rst

```diff
@@ -0,0 +1,150 @@
+
+License
+=======
+
+alertmanager_ is licensed under the Apache License 2.0.
+
+All relevant legal information can be found here
+
+  * https://github.com/prometheus/alertmanager/blob/master/LICENSE
+
+Prerequisites
+=============
+
+We need to prepare a couple of directories.
+
+The first directory is for storing the notification log and the alert silences:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/var/lib/alertmanager
+ [isabell@stardust ~]$
+
+The second directory is for storing the configuration files:
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/etc/alertmanager
+ [isabell@stardust ~]$
+
+
+Installation
+============
+
+Step 1
+------
+
+Find the latest version of alertmanager_ for the operating system ``linux`` and the architecture ``amd64`` from the `download page <https://prometheus.io/download>`_, download and extract it and enter the extracted directory:
+
+::
+
+ [isabell@stardust ~]$ wget https://github.com/prometheus/alertmanager/releases/download/v0.19.0/alertmanager-0.19.0.linux-amd64.tar.gz
+ [isabell@stardust ~]$ tar xvzf alertmanager-0.19.0.linux-amd64.tar.gz
+ [isabell@stardust ~]$ cd alertmanager-0.19.0.linux-amd64
+ [isabell@stardust alertmanager-0.19.0.linux-amd64]$
+
+Step 2
+------
+
+Move the binary to ``~/bin`` and the configuration file to ``~/etc/alertmanager``.
+
+::
+
+ [isabell@stardust alertmanager-0.19.0.linux-amd64]$ mv alertmanager ~/bin/
+ [isabell@stardust alertmanager-0.19.0.linux-amd64]$ mv alertmanager.yml ~/etc/alertmanager
+ [isabell@stardust alertmanager-0.19.0.linux-amd64]$
+
+Configuration
+=============
+
+Configure web server
+--------------------
+
+.. note::
+
+    alertmanager is running on port 9093.
+
+.. include:: includes/web-backend.rst
+
+Setup daemon
+------------
+
+Create the file ``~/etc/services.d/alertmanager.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:alertmanager]
+  command=alertmanager
+    --web.listen-address="127.0.0.1:9093"
+    --config.file=%(ENV_HOME)s/etc/alertmanager/alertmanager.yml
+    --storage.path=%(ENV_HOME)s/var/lib/alertmanager/
+  autostart=yes
+  autorestart=yes
+
+What the arguments for alertmanager_ mean:
+
+  * ``--web.listen-address``: The IP address and port alertmanager_ listens on.
+  * ``--config.file``: The full path to the alertmanager_ configuration file.
+  * ``--storage.path``: The path where alertmanager_ stores the notification log and the alert silences.
+
+Finishing installation
+======================
+
+Start alertmanager
+------------------
+
+.. include:: includes/supervisord.rst
+
+Now point your alert creating tool or your prometheus_ service to the configured alertmanager port and you should receive alerts sent out via the alertmanager.
+
+Best practices
+==============
+
+Security
+--------
+
+The alertmanager web interface MUST not be reachable from the public internet without authentication. It allows everyone to create and silence alarms which is probably not what you want.
```

Comment:
Then don't configure a web backend because then the web interface _will be_ reachable from the public internet. Leave out the whole `Configure web server` section.

Reply:
🤦‍♂

Thanks!